topology/m4/tone.m4

```diff
@@ -18,11 +18,13 @@ define(`W_TONE',
 `SectionVendorTuples."'N_TONE($1)`_tone_tuples_w" {'
 `	tokens "sof_tone_tokens"'
 `	tuples."word" {'
-`		SOF_TKN_TONE_SAMPLE_RATE'		ifdef(TONE_SAMPLE_RATE, STR(TONE_SAMPLE_RATE), "48000")
+`		SOF_TKN_TONE_SAMPLE_RATE'		ifdef(`TONE_SAMPLE_RATE', STR(TONE_SAMPLE_RATE), "48000")
 `	}'
 `}'
 `SectionData."'N_TONE($1)`_data_w" {'
 `	tuples "'N_TONE($1)`_tuples_w"'
+`}'
+`SectionData."'N_TONE($1)`_tone_data_w" {'
```

Comment:
I'm not sure why we need to define a BE here when the routing will describe all the BEs that tone is connected too. I was expecting to see more a a flag that would say to create a VFE (where any strings for VFE would come from w
tone widget)